src/sdk/sql_insert_row.h

```diff
@@ -125,10 +125,20 @@ class SQLInsertRows {
     inline uint32_t GetCnt() { return rows_.size(); }
     inline std::shared_ptr<SQLInsertRow> GetRow(uint32_t i) {
         if (i >= rows_.size()) {
-            return std::shared_ptr<SQLInsertRow>();
+            return {};
         }
         return rows_[i];
     }
+    inline const std::shared_ptr<hybridse::sdk::Schema> GetSchema() { return schema_; }
+    const std::vector<uint32_t> GetHoleIdx() {
```

Comment:
Where are the method used

Reply:
This change was for old insert prepared statement.
I leave it, cuz it's useful when we want get schema of rows. Before this change, we must create a new row to get. It's not good.